lib/util/viewer.js

```diff
@@ -211,8 +211,8 @@ export const getModeFromStop = (stop) => {
  * (in chronological order - 9:13am, 9:15am, etc.)
  */
 export function stopTimeComparator(a, b) {
-  const aTime = a.serviceDay + a.scheduledDeparture
-  const bTime = b.serviceDay + b.scheduledDeparture
+  const aTime = a.serviceDay + a.realtimeDeparture
```

Comment:
Do we know that `realtimeDeparture` is always present?

Reply:
Good catch